components/CDPUserForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { Alert, Form } from 'react-bootstrap'
+
+import { CDPUser, OnUpdateUser } from '../types/user'
+
+/**
+ * Form showing details for a specific CDP User.
+ */
+const CDPUserForm = ({
+  cdpUser,
+  isSelf,
+  onUpdateUser
+}: {
+  cdpUser: CDPUser
+  isSelf?: boolean
+  onUpdateUser: OnUpdateUser
```

Comment:
I think our preference now is to declare a Props interface.

Reply:
Good catch thanks! Refactored in ace74aefd17cc2d5aa0af0557d379e474cd8c1ae